Cartridges/int_checkoutcom_sfra/cartridge/scripts/helpers/eventsHelper.js

```diff
@@ -126,7 +126,7 @@ var eventsHelper = {
             paymentInstrument.paymentTransaction.setType(PaymentTransaction.TYPE_CAPTURE);
 
             setPaymentStatus(order);
-            order.setConfirmationStatus(1);
+            order.setConfirmationStatus(2);
```

Comment:
What does this number mean ? Could you use a constant which indicate some meaningful words